packages/core/index.d.ts

```diff
@@ -37,7 +37,7 @@ declare module '@rjsf/core' {
         name?: string;
         noHtml5Validate?: boolean;
         noValidate?: boolean;
-        ObjectFieldTemplate?: React.StatelessComponent<ObjectFieldTemplateProps>;
+        ObjectFieldTemplate?: React.FunctionComponent<ObjectFieldTemplateProps>;
```

Comment:
Are there issues with FunctionComponent (see https://github.com/facebook/create-react-app/pull/8177)? Or do those not apply here?

Reply:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/34e20b805043483182fb77a96008244a70874dd2/types/react/v17/index.d.ts#L542

children are now optional as i understand better in line 822
```type PropsWithChildren<P> = P & { children?: ReactNode | undefined };```

Reply:
> Also it would be worth to extract TitleFieldProps and DescriptionFieldProps types.

I am doing that as part of a v5 change coming in about 1.5 months

Reply:
> So it would be nice to add optional children prop for backward compatibility + more powerful extensebility.

Thanks. Can you add a PR that does this?